cloudflare/resource_cloudflare_access_mutual_tls_certificate.go

```diff
@@ -0,0 +1,195 @@
+		Delete: resourceCloudflareAccessMutualTLSCertificateDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareAccessMutualTLSCertificateImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"account_id": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Computed:      true,
+				ConflictsWith: []string{"zone_id"},
+			},
+			"zone_id": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Computed:      true,
+				ConflictsWith: []string{"account_id"},
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"certificate": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"associated_hostnames": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"fingerprint": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceCloudflareAccessMutualTLSCertificateCreate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*cloudflare.API)
+
+	newAccessMutualTLSCertificate := cloudflare.AccessMutualTLSCertificate{
+		Name:        d.Get("name").(string),
+		Certificate: d.Get("certificate").(string),
+	}
+	newAccessMutualTLSCertificate.AssociatedHostnames = expandInterfaceToStringList(d.Get("associated_hostnames"))
+
+	log.Printf("[DEBUG] Creating Cloudflare Access Mutual TLS certificate from struct: %+v", newAccessMutualTLSCertificate)
+
+	identifier, err := initIdentifier(d)
+	if err != nil {
+		return err
+	}
+
+	var accessMutualTLSCert cloudflare.AccessMutualTLSCertificate
+	if identifier.Type == AccountType {
+		accessMutualTLSCert, err = client.CreateAccessMutualTLSCertificate(identifier.Value, newAccessMutualTLSCertificate)
+	} else {
+		accessMutualTLSCert, err = client.CreateZoneAccessMutualTLSCertificate(identifier.Value, newAccessMutualTLSCertificate)
+	}
+	if err != nil {
+		return fmt.Errorf("error creating Access Mutual TLS Certificate for %s %q: %s", identifier.Type, identifier.Value, err)
+	}
+
+	d.SetId(accessMutualTLSCert.ID)
+
+	return resourceCloudflareAccessMutualTLSCertificateRead(d, meta)
+}
+
+func resourceCloudflareAccessMutualTLSCertificateRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*cloudflare.API)
+
+	identifier, err := initIdentifier(d)
+	if err != nil {
+		return err
+	}
+
+	var accessMutualTLSCert cloudflare.AccessMutualTLSCertificate
+	if identifier.Type == AccountType {
+		accessMutualTLSCert, err = client.AccessMutualTLSCertificate(identifier.Value, d.Id())
+	} else {
+		accessMutualTLSCert, err = client.ZoneAccessMutualTLSCertificate(identifier.Value, d.Id())
+	}
+
+	if err != nil {
+		if strings.Contains(err.Error(), "HTTP status 404") {
+			log.Printf("[INFO] Access Mutal TLS Certificate %s no longer exists", d.Id())
+			d.SetId("")
+			return nil
+		}
+		return fmt.Errorf("Error finding Access Mutual TLS Certificate %q: %s", d.Id(), err)
```

Comment:
let's lowercase the exception message to align with Go's conventions

```suggestion
		return fmt.Errorf("error finding Access Mutual TLS Certificate %q: %s", d.Id(), err)
```